content/en/platform/corda/4.9/enterprise/notary/hsm-support.md

```diff
@@ -61,7 +61,7 @@ Add the following entries to your workers node.conf files, replacing the placeho
 ```sh
 cryptoServiceName: "AZURE_KEY_VAULT"
 cryptoServiceConf: "<path to the crypto service configuration file>"
-cryptoServiceTimeout: "<the desired timeout, default: 1s>"
+cryptoServiceTimeout: "<the desired timeout, default: 10s>"
```

Comment:
shouldn't this be consistent with the previous docs and state 10000 msec ?

Reply:
Thanks Yariv, done.